plugins/synaptics-prometheus/meson.build

```diff
@@ -31,8 +31,9 @@ shared_module('fu_plugin_synaptics_prometheus',
 )
 
 if get_option('tests')
-  testdatadir = join_paths(meson.current_source_dir(), 'data')
-  cargs += '-DTESTDATADIR="' + testdatadir + '"'
+  promtest = environment()
```

Comment:
not testdatadirs?

Reply:
I was running into problems re-using the same variable across files..

Reply:
unexpected. I know there's only a "global scope" in meson, but i don't think re-defining should affect things...

Reply:
Well I think it was an order of events thing.  Meson builds up all the variables, compiles things, and then runs things.  Since these variables are very specific to a specific directory failures ensued.

Reply:
I think then we should do something consistent for all the plugins. Perhaps `prometheus_env` or something?